tests/vega_sim/api/test_data_raw.py

```diff
@@ -1,6 +1,7 @@
 import grpc
 import pytest
 from concurrent.futures import ThreadPoolExecutor
+from datetime import datetime
```

Comment:
Could you change these to just `import datetime`? It's the pattern we've used elsewhere so good to keep consistent :)